scripts/retrieve_training_metrics.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
When you pass a date, the script should get all tasks from that date forward (so we can later generate a graph).

Reply:
Maybe we can implement that later, when we have at least a few tasks indexed by date. Could you file an issue?

Reply:
Sure

Reply:
https://github.com/mozilla/bugbug/issues/614